### PR TITLE
feat: Add Oracle dialect specific documentation v6

### PR DESCRIPTION
**Pull request by sudarshan12s**

### CODE REVIEW

File: versioned_docs/version-6.x.x/getting-started.md [hunk 1]
```diff
@@ -44,7 +45,7 @@ const sequelize = new Sequelize({
 // Option 3: Passing parameters separately (other dialects)
 const sequelize = new Sequelize('database', 'username', 'password', {
   host: 'localhost',
-  dialect: /* one of 'mysql' | 'mariadb' | 'postgres' | 'mssql' */
+  dialect: /* one of 'mysql' | 'mariadb' | 'postgres' | 'mssql' | 'Oracle' */
```
Comment (cjbj) [inline versioned_docs/version-6.x.x/getting-started.md:48]: Lower case 'oracle'?

File: versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md [hunk 2]
```diff
@@ -199,6 +199,31 @@ For running integration test:
 SEQ_ACCOUNT=myAccount SEQ_USER=myUser SEQ_PW=myPassword SEQ_ROLE=myRole SEQ_DB=myDatabaseName SEQ_SCHEMA=mySchema SEQ_WH=myWareHouse npm run test-integration-snowflake
 ```
 
+### Oracle Database
+
+The underlying connector library used by Sequelize for Oracle is the [node-oracledb](https://www.npmjs.com/package/oracledb) package.  
+See [Releases](/releases#oracle-support-table) to see which versions of Oracle Database & node-oracledb are supported.
+
+node-oracledb needs [instant client library files](https://www.oracle.com/database/technologies/instant-client/downloads.html) to work. You can use node-oracledb [quick start](https://oracle.github.io/node-oracledb/INSTALL.html#quickstart) link for installations.
```
Comment (cjbj) [inline versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md:207]: Initcap IC and say 'Oracle Instant Client'.

File: versioned_docs/version-6.x.x/getting-started.md (hunk 1 above)
Comment (sudarshan12s) [reply versioned_docs/version-6.x.x/getting-started.md]: Done

File: versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md (hunk 2 above)
Comment (sudarshan12s) [reply versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md]: modified to 

> IC [Oracle Instant Client]

**Comment by sdepold** [comment]
I might be wrong but it seems to me that this is also changing v7 docs right? I think it would be cool if we only change main website and the v6 docs for now and when oracle hits v7, then we extend the v7 docs too

**Comment by cjbj** [comment]
This appears to be superseded by https://github.com/sequelize/website/pull/261

**Comment by sudarshan12s** [comment]
> I might be wrong but it seems to me that this is also changing v7 docs right? I think it would be cool if we only change main website and the v6 docs for now and when oracle hits v7, then we extend the v7 docs too

Yes It had changes for main too.  I have removed the v7 docs and made a new [PR](https://github.com/sequelize/website/pull/261).

**Comment by sdepold** [comment]
Thanks :)

**Pull request closed without merging**